src/com/lightdev/app/shtm/ParaStyleDialog.java

```diff
@@ -247,7 +247,7 @@ private void initContentTypes() {
         cTypes[i++] = new NamedObject(HTML.Tag.H6.toString(), Util.getResourceString("cTagNameHead6"));
         cTypes[i++] = new NamedObject(HTML.Tag.A.toString(), Util.getResourceString("cTagNameLink"));
         cTypes[i++] = new NamedObject(HTML.Tag.UL.toString(), Util.getResourceString("cTagNameUL"));
-        cTypes[i++] = new NamedObject(HTML.Tag.OL.toString(), Util.getResourceString("cTagNameOL"));
+        cTypes[i] = new NamedObject(HTML.Tag.OL.toString(), Util.getResourceString("cTagNameOL"));
```

Comment:
Keeping i++ makes sense in case someone continues the list. I do not think this optimization makes sense but rather think it is dangerous.